app/src/main/java/be/chvp/nanoledger/ui/util/Quadruple.kt

```diff
@@ -0,0 +1,8 @@
+package be.chvp.nanoledger.ui.util
+
+import java.io.Serializable
+
+data class Quadruple<A, B, C, D>(var first: A, var second: B, var third: C, var fourth: D) :
+    Serializable {
+    override fun toString(): String = "($first, $second, $third, $fourth)"
+}
```

Comment:
To be honest, the use of `Triple` was already something I wasn't too happy with. If we're introducing a new class for this anyway, I'd prefer it to be clearer about its contents. Maybe something like `EditablePosting`, with `account`, `currency`, `amount` and `note` fields.